app/src/main/java/de/tum/in/tumcampusapp/managers/TransportManager.java

```diff
@@ -407,20 +399,42 @@ public StationResult(String station, String id, int quality) {
     }
 
     public static class WidgetDepartures {
```

Comment:
ok i extracted all the nested classes from the transport manager to a model package called efa